staging/src/k8s.io/apiextensions-apiserver/test/integration/table_test.go

```diff
@@ -146,7 +148,7 @@ func TestTableGet(t *testing.T) {
 	}
 	t.Logf("%v table list: %#v", gvk, tbl)
 
-	if got, expected := len(tbl.ColumnDefinitions), 5; got != expected {
+	if got, expected := len(tbl.ColumnDefinitions), 6; got != expected {
 		t.Errorf("expected %d headers, got %d", expected, got)
 	} else {
 		alpha := metav1beta1.TableColumnDefinition{Name: "Alpha", Type: "string", Format: "", Description: "Custom resource definition column (in JSONPath format): .spec.alpha", Priority: 0}
```

Comment:
The alpha field in the CRD doesn't have the description but this does. How are both of them equal then?

Reply:
it's defaulted.